deploy.py

```diff
@@ -528,6 +528,10 @@ def check_prerequisites(self):
       assert_mandatory_binary("nrfutil")
       assert_python_library("intelhex")
       assert_python_library("nordicsemi.lister")
+      nrfutil_version = __import__("nordicsemi").version.NRFUTIL_VERSION
+      if not nrfutil_version.startswith("6."):
+        fatal(("You need to install nrfutil python3 package v6.0 or above. "
+               "Found: {}".format(nrfutil_version)))
```

Comment:
What if no version is found? Shouldn't the message say that it's not found?

Reply:
I haven't checked earlier versions but at least since v1.2.4 of the package this fields exists.
And there's a check earlier that the package "nordicsemi" exists and is imported successfully.

Reply:
> And there's a check earlier that the package "nordicsemi" exists and is imported successfully.

Sounds good then!